docs/CONTRIBUTING.md

```diff
@@ -108,36 +109,105 @@ bibtex: # citation(s) for this dataset in BibTeX format
+Question: Is the molecule with the {SMILES__description} representation of {SMILES#} {mutagenic__names__adjective}?
+Options:
+{mutagenic%}
+Answer: {%multiple_choice_result}
+```
+Example output:
+```
+Task: Please answer the multiple choice question below with A or B.
+Question: Is the molecule with the SMILES representation of CC(C)NCC(O)c1ccc2ccccc2c1 Ames mutagenic?
+Options:
+A) False
+B) True
+Answer: A"
+```
 
-```yaml
-templates:
-  - prompt: "Please answer the following chemistry question.\nDerive for the molecule with the <molecule#text> <molecule#value> the <expt_value#text>."
-    completion: "<exp_value.value>"
-  - prompt: "Please answer the following question.\nPredict the <expt_value#text> for <molecule#value>."
-    completion: "<exp_value#value>"
-fields:
-  exp_value:
-    values:
-      - name: lab_value
-        column: lab_value
-        text: adsorption energy
-  molecule:
-    values:
-      - name: smiles
-        column: smiles
-        text:
-      - name: inchi
-        column: inchi
-        text: InChI
+##### Template for benchmarking
+```
+Task: Please answer the multiple choice question below with {%multiple_choice_enum%2%aA1}.
+Question: Is the molecule with the {SMILES__description} representation of {SMILES#} {mutagenic__names__adjective}?
+Options:
+{mutagenic%}
+Answer:<EOI> {%multiple_choice_result}
+```
+The benchmarking setup exports additional fields for the benchmarking setup, see the example below:
+`{"input":"Task: Please answer the multiple choice question below with 1 or 2.\nQuestion: Is the molecule with the SMILES representation of BrCBr Ames mutagenic?\nOptions:\n1.) False\n2.) True\nAnswer:","output":" 2","output_choices":["1","2"],"correct_output_index":"1"}`
+Please have a look at the following section below about the general benchmarking template setup.
+
+#### Example text templates 4 for flexible multiple choice setups
+More flexible multiple choice setups are also supported. The standard multiple choice setup from "Example text templates 3 for multiple choice setups" is intended for features of molecules as those are deduplicated during the sampling process. In contrast, this flexible multiple choice setup also lets you use the molecule identifiers, e.g., SMILES, in the multiple choice options.
+
+For this we only need to add one component to the previously outlined multiple choice format:
+* In order to let the model predict which `SMILES` has or has not the boolean variable `penetrate_BBB` we simply add `SMILES%penetrate_BBB%` as an enumeration placeholder for the possible options. With that the list of the multiple choice enumerations shows the SMILES data. Note that the `penetrate_BBB#not &NULL` is needed because the sampling is based on the individual row and depending on if `penetrate_BBB` is `True` or `False` we look for a different label because in the code we compare the sampled options to the `penetrate_BBB` value of the row.
```

Comment:
>Note that the `penetrate_BBB#not &NULL` is needed because the sampling is based on the individual row and depending on if `penetrate_BBB` is `True` or `False` we look for a different label because in the code we compare the sampled options to the `penetrate_BBB` value of the row.

Was not very clear to me. Perhaps it helps to add somewhere a clarification of what 
- row
- label 
- column 

mean (with example) in this contribution guide

Reply:
tried to make it more clear